server/controllers/process.go

```diff
@@ -165,7 +167,7 @@ func ProcessRun(rw io.ReadWriteCloser, c *api.Context) error {
 
 	ev, err := url.ParseQuery(c.Header("Environment"))
 	if err != nil {
-		return helpers.CodeError(rw, -1, err)
+		return helpers.CodeError(rw, 255, err)
```

Comment:
Believe the thought process of using `-1` was to distinguish between an internal error and process exit code. Exit codes < 0 are not valid so praxis would use negative exit codes as to not clash with a process.